recipe/meta.yaml

```
@@ -1,13 +1,13 @@
 {% set name = "zstash" %}
-{% set version = "1.4.4rc2" %}
+{% set version = "1.5.0rc1" %}
 
```

@tomvothecoder might have some other idea of how to do this. There were build errors trying to build with python `3.10`, which this will fix. 

```suggestion
{% set python_min = "3.11" %}
```

---

This is expected.  Python <3.11 support has been disabled, right?

---

Yes, `zstash` now supports `python >=3.11,<3.14`. Prior to this line, there was a CI error related to trying to build with python 3.10. If there's a better way to enforce the min version, LMK.

---

This is the correct way.

---

Looks good